src/libraries/System.Private.CoreLib/src/System/IO/Strategies/AsyncWindowsFileStreamStrategy.ValueTaskSource.cs

```diff
@@ -108,6 +107,8 @@ internal void RegisterForCancellation(CancellationToken cancellationToken)
 
             internal virtual void ReleaseNativeResource()
```

Comment:
it does not need to be `virtual` anymore

```suggestion
            internal void ReleaseNativeResource()
```